home/javascript/react/containers/FigureEdit.js

```diff
@@ -22,6 +22,51 @@ function FigureEdit({ pubId }) {
             });
     }, [pubId]);
 
+    useLayoutEffect(() => {
```

Comment:
Can you not use useEffect()? useLayoutEffect() can slow down rendering...

Reply:
Updated